motion/moran/java_to_ruby.rb

```diff
@@ -34,9 +34,9 @@ def hashmap_to_hash(hm)
       end
     end
   end
-  
+
   def json_object_to_hash(json_obj)
-    return nil if json_obj.nil?
+    return nil if (json_obj.nil? || json_obj.toString == "null")
```

Comment:
null values from a server response become "java.lang.JSONObjects$1". Running `toString` on that returns "null" so we can turn that back to nil.